server/intelligence-service/app/mentor/response.py

```
@@ -0,0 +1,220 @@
+"""
+LangGraph-based response generation with proper streaming using astream_events.
+"""
+import json
+from typing import AsyncGenerator, Dict, Any
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)